wrangler-core/src/main/java/io/cdap/directives/datamodel/DataModelMapColumn.java

```diff
@@ -151,7 +152,7 @@ public void initialize(Arguments args) throws DirectiveParseException {
   @Override
   public List<Row> execute(List<Row> rows, ExecutorContext context) throws DirectiveExecutionException {
     for (Row row : rows) {
-      ColumnConverter.convertType(NAME, row, column, targetFieldTypeName);
+      ColumnConverter.convertType(NAME, row, column, targetFieldTypeName, null, RoundingMode.UNNECESSARY);
```

Comment:
Do you really need to change it? If so, add a test. I am not sure what `args.contains("scale")` would return and if you would get an NPE.
BTW: It would be nice to handle null scale as well same as no scale

Reply:
Setting it as null is alright because convertType is a utility method where setting scale as null is handled anyways. In any case, args.contains("scale") returns false when scale is null (ex, when scale isn't specified in the directive)